src/components/Bridge/BridgeCard/index.tsx

```diff
@@ -181,6 +181,7 @@ const BridgeCard: React.FC<BridgeCardProps> = (props) => {
           method: 'wallet_switchEthereumChain',
           params: [{ chainId: `0x${chain?.chain_id?.toString(16)}` }],
         });
+        window.location.reload();
```

Comment:
we are using same code here `src/components/NetworkSelection/NetworkSelection.tsx`, can we reuse it ?